### PR TITLE
Add favicon to dev server and test

### PR DESCRIPTION
**Pull request by vilsbole**
[ch6504](https://app.clubhouse.io/mycrypto/story/6504/add-favicon-to-devserver)

### CODE REVIEW

File: __tests__/website.test.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { Selector, ClientFunction } from 'testcafe';
+import request from 'request';
```
Comment (FrederikBolding) [inline __tests__/website.test.js:2]: Remove unused import

File: __tests__/website.test.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+import { PAGES } from './fixtures';
+import NoAccountsPage from './no-accounts-page.po';
+
+const noAccounts = new NoAccountsPage();
+const getHost = ClientFunction(() => window.location.host);
```
Comment (FrederikBolding) [inline __tests__/website.test.js:8]: Remove unused const

File: __tests__/website.test.js (hunk 1 above)
Comment (vilsbole) [reply __tests__/website.test.js]: Done

File: __tests__/website.test.js (hunk 2 above)
Comment (vilsbole) [reply __tests__/website.test.js]: Done

**Pull request merged**